pkg/plugins/policies/meshtls/plugin/v1alpha1/plugin.go

```diff
@@ -284,7 +284,7 @@ func configureListener(
 	getName := naming.GetNameOrFallbackFunc(unifiedNaming)
 
 	inboundID := kri.WithSectionName(kri.From(proxy.Dataplane), iface.WorkloadPort).String()
-	inboundContextualID := naming.MustContextualInboundName(proxy.Dataplane, iface.WorkloadPort)
+	inboundContextualID := naming.MustContextualInboundName(proxy.Dataplane, inbound.GetSectionName())
```

Comment:
shouldn't we use `iface.InboundName` here?
```suggestion
	inboundContextualID := naming.MustContextualInboundName(proxy.Dataplane, iface.InboundName)
```